tests/kernels/test_cutlass_moe.py

```diff
@@ -242,3 +266,113 @@ def test_cutlass_moe_cuda_graph(
                                    cutlass_output,
                                    atol=9e-2,
                                    rtol=1e-2)
+
+
+@pytest.mark.parametrize("m,n,k", MNK_FACTORS)
+@pytest.mark.parametrize("e", NUM_EXPERTS)
+@pytest.mark.parametrize("topk", TOP_KS)
+@pytest.mark.parametrize("dtype", [torch.bfloat16, torch.half])
+@pytest.mark.skipif(
+    (lambda x: x is None or not ops.cutlass_group_gemm_supported(x.to_int()))(
+        current_platform.get_device_capability()),
+    reason="Grouped gemm is not supported on this GPU type.")
+def test_cutlass_moe_16_bit_no_graph(
+    m: int,
+    n: int,
+    k: int,
+    e: int,
+    topk: int,
+    dtype: torch.dtype,
+):
+    current_platform.seed_everything(7)
+    with set_current_vllm_config(
+            VllmConfig(parallel_config=ParallelConfig(
+                pipeline_parallel_size=1))):
+
+        a = torch.ones((m, k), device="cuda", dtype=dtype) / 10
+        w1 = torch.randn(
+            (e, 2 * n, k), device="cuda", dtype=dtype).transpose(1, 2) / 10
+        w2 = torch.randn(
+            (e, k, n), device="cuda", dtype=dtype).transpose(1, 2) / 10
+
+        ab_strides1 = torch.full((e, ), k, device="cuda", dtype=torch.int64)
+        c_strides1 = torch.full((e, ), 2 * n, device="cuda", dtype=torch.int64)
+        ab_strides2 = torch.full((e, ), n, device="cuda", dtype=torch.int64)
+        c_strides2 = torch.full((e, ), k, device="cuda", dtype=torch.int64)
+
+        score = torch.randn((m, e), device="cuda", dtype=dtype)
+        topk_weights, topk_ids = fused_topk(a, score, topk, renormalize=False)
+
+        triton_output = fused_experts(a, w1.transpose(1,
+                                                      2), w2.transpose(1, 2),
+                                      topk_weights, topk_ids)
+        cutlass_output = cutlass_moe(a, w1, w2, topk_weights, topk_ids,
+                                     ab_strides1, c_strides1, ab_strides2,
+                                     c_strides2)
+
+        print(triton_output)
+        print(cutlass_output)
+        print("*")
```

Comment:
nit: Do we need this prints?

Reply:
The tolerances in the tests are a bit high, so I use these prints to examine manually how off the values are if I'm close to the treshold

Reply:
I think the prints are fine for debugging but I don't think we should push with them enabled.